rocAL/rocAL_pybind/amd/rocal/plugin/tf.py

```diff
@@ -209,21 +169,3 @@ def __init__(self,
         pipe = pipelines
         super(ROCALIterator, self).__init__(pipe, tensor_layout = pipe._tensor_layout, tensor_dtype = pipe._tensor_dtype,
                                                             multiplier=pipe._multiplier, offset=pipe._offset, device=device, device_id=device_id)
```

Comment:
Add EOF

Reply:
As seen in the video pipeline example, the file as EOF, but its not visible in the file changes tab